user-stories.md

```diff
@@ -0,0 +1,48 @@
+Movie Recommendations:
+
+As a movie fan, I want to discover and manage a personalized collection of recommended films.
+
+Feature 1: Enter in to the Movie Collection
+User Story: When I open the app, I want to click a button on the homepage that shows me a gallery of movie posters. Then, when I click on a movie, the poster flips to reveal a brief description on the back.
```

Comment:
> Then, when I click on a movie, the poster flips to reveal a brief description on the back.

If you're going to have a page that shows the details, consider skipping this for now.